templates/web/components/messages.html

```diff
@@ -19,7 +19,7 @@
           Toast.fire({
             icon: '{{ message.tags }}',
             title: '{{ message.tags|capfirst }}',
-            text: '{{ message }}'
+            text: '{{ message|escape }}'
```

Comment:
curious what you observed that resulted in this

Reply:
Oh this was a quick thing I tried, because I saw this:
![Screenshot from 2024-04-02 13-56-00](https://github.com/dimagi/open-chat-studio/assets/64970009/f3e5d24f-9e3a-4df2-af1c-de23ccad446d)

I added the `escape` filter but sort of forgot about this and didn't test it again. Not only should I not have escaped it (should have used `|safe` or something), it didn't fix the issue. I'll just remove the escape filter and fix this in a separate PR. Thanks for asking.